### PR TITLE
Build: Add config for browserstack-runner

### PR DESCRIPTION
**Pull request by jzaefferer**
This is a work in progress, to debug the issues we're still having with this setup.

### CODE REVIEW

**Comment by jzaefferer** [comment]
Updated to include both unit and functional tests and a separate config for IE8. Can run the latter with:

```sh
BROWSERSTACK_JSON=browserstack-ie8.json browserstack-runner
```

Both ran successfully on my local machine. I'm still working on the Travis integration.

File: browserstack-ie8.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "test_framework": "qunit",
+  "test_path": ["test/unit-es5-shim.html", "test/functional-es5-shim.html"],
```
Comment (rxaviers) [inline browserstack-ie8.json]: Space `[ "test/unit-es5-shim.html", "test/functional-es5-shim.html" ]`?

File: browserstack.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "test_framework": "qunit",
+  "test_path": ["test/unit.html", "test/functional.html"],
```
Comment (rxaviers) [inline browserstack.json]: Space `test_path": [ "test/unit.html", "test/functional.html" ],`?

**Comment by jzaefferer** [comment]
Btw. since the Travis integration is still problematic, we can start by only landing the config files, so that we can use them locally. We can land the other changes later.

**Comment by rxaviers** [comment]
Sounds good to me.

**Comment by rxaviers** [comment]
Sounds good to me. Do you want to update this branch accordingly and potentially creating a new PR to track Travis integration?

**Comment by rxaviers** [comment]
> Can run the latter with:
> `BROWSERSTACK_JSON=browserstack-ie8.json browserstack-runner`

This is really cool!

**Pull request merged**